params/Augustinus vourinus.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4
+    
+# e.g. "anas_krystallinus"    
+species.name <- ""anas_krystallinus" "
```

Comment:
The double quotes here will cause an error
```suggestion
species.name <- "anas_krystallinus"
```